README.md

```diff
@@ -101,66 +93,62 @@ The following are the samples.
 
 #### Rosbag
 
-1. Download sample map from [here](https://drive.google.com/open?id=1ovrJcFS5CZ2H51D8xVWNtEvj_oiXW-zk).
-
-2. Download sample rosbag from [here](https://drive.google.com/open?id=1BFcNjIBUVKwupPByATYczv2X4qZtdAeD).
-3. Launch Autoware
+1. Download sample pointcloud and vector maps from [here](https://drive.google.com/open?id=1ovrJcFS5CZ2H51D8xVWNtEvj_oiXW-zk).
+2. Download a sample rosbag from [here](https://drive.google.com/open?id=1BFcNjIBUVKwupPByATYczv2X4qZtdAeD).
+3. Open a terminal and launch Autoware
 
 ```sh
 cd AutowareArchitectureProposal
 source install/setup.bash
 roslaunch autoware_launch logging_simulator.launch map_path:=[path] vehicle_model:=lexus sensor_model:=aip_xx1 rosbag:=true
 ```
 
-\* Absolute path is required for map_path.
+> Note that the absolute path is required for `map_path` above.
 
-4. Play rosbag
+4. Open a second terminal and play the rosbag file
 
 ```sh
 rosbag play --clock [rosbag file] -r 0.2

```

Comment:
Again using the convention `/path/to/rosbag.bag`. This identifies that it is an absolute path and that we should be referencing the actual bag itself in the path rather than the folder it is in, for example.